home-data-migration/src/main/java/com/softserveinc/ita/homeproject/homedatamigration/migrations/LiquibaseUpdate.java

```diff
@@ -0,0 +1,27 @@
+package com.softserveinc.ita.homeproject.homedatamigration.migrations;
+
+import liquibase.Contexts;
+import liquibase.LabelExpression;
+import liquibase.Liquibase;
+import liquibase.database.jvm.JdbcConnection;
+import liquibase.exception.LiquibaseException;
+import liquibase.integration.commandline.CommandLineResourceAccessor;
+import liquibase.resource.CompositeResourceAccessor;
+import org.postgresql.Driver;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class LiquibaseUpdate {
+
+    final static String PATH = "db/changelog/db.changelog-master.xml";
```

Comment:
What is about the access modifier, can it be private?

Reply:
I think yes. Made it private

Reply:
But it should remain as a constant:
`private final static ... UPPERCASE_NAME_OF_VARIABLE = ....;`